cv_next/server/api/cvs.ts

```diff
@@ -147,13 +147,18 @@ export async function getPaginatedCvs(
     const supabase = SupabaseHelper.getSupabaseInstance();
     let query = supabase
```

Comment:
why this is not one query with join?

Reply:
IMHO it should be a seperate query as we;re querying different data. I've been working on a function to query IDs from fullnames seperately